### PR TITLE
Transaction Diff

### PR DESCRIPTION
**Pull request by matbme**
Adds the diffcommand for listing added/removed/modified files between transaction. This PR also adds [pterm](https://github.com/pterm/pterm) as a dependency.

Blocked by pterm since it has no deb package yet.

### CODE REVIEW

**Comment by matbme** [comment]
Orchid is missing the following pterm bindings required by `TransactionDiff`:
- `BulletListItem`
- `NewStyle`
- `DefaultBulletList`
- Color and font types (e.g. `Bold`, `FgYellow`, `FgGreen`, ...)

File: core/transaction.go [hunk 1]
```diff
@@ -187,6 +190,92 @@ func ApplyTransaction() error {
 	return nil
 }
 
+// TransactionDiff prints a list of added, modified, and removed files
+// from the lastest transaction.
+func TransactionDiff() {
+	PrintVerbose("step:  TransactionDiff")
+	if !AreTransactionsLocked() {
+		cmdr.Warning.Println("No transaction has been made since last reboot. Nothing to diff.")
```
Comment (bketelsen) [inline core/transaction.go:198]: let's file an issue to capture the inability to localize in the library code

**Review by bketelsen — APPROVED** [review]
LGTM

File: core/transaction.go (hunk 1 above)
Comment (bketelsen) [reply core/transaction.go:198]: https://github.com/Vanilla-OS/orchid/issues/12

**Pull request merged**